src/app/features/profile/profile.page.html

```diff
@@ -14,21 +14,35 @@
       <div mat-line>NUM {{ t('balance') }}</div>
       <div mat-line>ERC20</div>
       <div class="num-balance">
-        <ng-container *ngIf="networkConnected$ | ngrxPush; else offline">
-          $ {{ ethNumBalance$ | ngrxPush | number: '1.2-2' }}
+        <ng-container
+          *ngIf="networkConnected$ | ngrxPush; else balance_offline"
+        >
+          <ng-container
+            *ngIf="!(isLoadingBalance$ | ngrxPush); else balance_loading"
+          >
+            $ {{ ethNumBalance$ | ngrxPush | number: '1.2-2' }}
+          </ng-container>
         </ng-container>
-        <ng-template #offline> --offline-- </ng-template>
+        <ng-template #balance_offline> --offline-- </ng-template>
+        <ng-template #balance_loading>
```

Comment:
We should use camel case naming for consistency.

Reply:
Good catch!